latest.py

```diff
@@ -1,134 +1,60 @@
-#----------------------------------------------------------------------
-#   
-#   Project:            New goto funtion
-#   Description:        Just point the target coordinates
-#                       
-#   Date:               03.12.2021
-#   Maximum score:      70 points
-#   Time left:          31 seconds
-#
-#   Starting position:  A
-#   Orientation:        Facing East (standard, 0 for the robot)
-#   Preload:            Zero Rings
-#
-#----------------------------------------------------------------------
+# tiny 70 points - 06.12.2021
 
-# Library imports
 from vexcode_vrc import *
 from math import sqrt
 
-def goto(target_x, target_y, reverse):
-    global angle, x1, y1, x2, y2, delta_x, delta_y, head, distance, directio
-    angle = 0
-    x1 = gps.x_position(MM)
-    y1 = gps.y_position(MM)
-    x2 = target_x
-    y2 = target_y
-    head = gps.heading()   # north is 0, but for the robot 0 is east - from start
-    delta_x = x2 - x1
-    delta_y = y2 - y1
-    distance = math.sqrt(delta_x**2 + delta_y**2)     # pythagorean theorem
-    if ( delta_x ):
-        directio = - math.atan(delta_y / delta_x) * 180 / math.pi
-    else:
-        directio = 90
-
-    if ( delta_x < 0 ):
-        directio = directio + 180
-
-    monitor_variable("angle","x1","y1", "x2", "y2", "delta_x", "delta_y","head", "distance", "directio")
-    drivetrain.turn_to_heading(directio, DEGREES, wait=True)
-    if (reverse == 0):
-        drivetrain.drive_for(FORWARD, distance, MM, wait=True)
-    else:
-        drivetrain.drive_for(REVERSE, -distance, MM, wait=True)
-
-def move(direction, coordinate, reverse):
-    global angle, x, y, head, dist
-    angle = 0
-    x = gps.x_position(MM)
-    y = gps.y_position(MM)
-    head = gps.heading()   # north is 0, but for the robot 0 is east - from start    
-    monitor_variable("angle","dist","x","y","head") 
-    if ( direction == "right" ):
-        angle = 0
-        dist = coordinate - gps.x_position(MM)
-    if ( direction == "left" ):
-        angle = -180
-        dist = gps.x_position(MM) - coordinate
-    if ( direction == "up" ):
-        angle = -90 # 270
-        dist = coordinate - gps.y_position(MM)
-    if ( direction == "down" ):
-        angle = 90
-        dist = gps.y_position(MM) - coordinate
-    drivetrain.turn_to_heading(angle, DEGREES, wait=True)
-    if (reverse == 0):
-        drivetrain.drive_for(FORWARD, dist, MM, wait=True)
-    else:
-        drivetrain.drive_for(REVERSE, -dist, MM, wait=True)
```

Comment:
Thanks for renaming dist to distance

And fixing the -dist in REVERSE, since direction of robot is already adjusted by 180 degrees